htdocs/compta/tva/index.php

```diff
@@ -88,7 +90,7 @@ function pt ($db, $sql, $date)
             print '<td class="nowrap">'.$obj->dm."</td>\n";
             $total = $total + $obj->mm;
 
-            print '<td class="nowrap" align="right">'.price($obj->mm)."</td><td >&nbsp;</td>\n";
+            print '<td class="nowrap" align="right">'.price(price2num($obj->mm,1))."</td><td >&nbsp;</td>\n";
```

Comment:
Ok I have the problem of rounding only on some database, I try to find the cause